### PR TITLE
Added simple error checking and associated error states.

### PR DESCRIPTION
**Pull request by alexmorris**
Added password/cleartext toggle
Added user feedback for error
Added compass plugin for auto prefixing and minification

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -44,6 +44,8 @@
     {% endblock foundation_js %}
 
     {# Custom JS #}
+    <script src="{% static "js/hideShowPassword.min.js" %}"></script>
```
Comment (dhilton) [inline templates/base.html]: Do you want this file on all pages?

Comment (alexmorris) [inline templates/base.html]: nope good point, not sure how to add page specific scripts though ;)

Comment (dhilton) [inline templates/base.html]: In the index file, you can override the block (as that inherits from base) and call block.super to pull in anything from the base.html. That way the js only gets pulled in on the sign page.

Comment (alexmorris) [inline templates/base.html]: have made amends inside my current branch, the next stories output will also address that issue

**Pull request merged**